.gitignore

```diff
@@ -1,4 +1,5 @@
 *.pyc
+*.py
```

Comment:
wait, aren't these normal python files?  Shouldn't it be `*.pyo`?

Reply:
ugh. yeah. fixed that. not sure what happened there